docs/Importing-Parquet-and-ZST.md

```diff
@@ -21,7 +20,7 @@ that can be easily imported into Brim.
 
 # Example
 
-Preprocessing can be performed with the [`zq`](https://github.com/brimdata/zed/tree/main/cmd/zq)
+Preprocessing can be performed with the [`zq`](https://github.com/brimdata/zed/blob/main/docs/zq/README.md)
```

Comment:
```suggestion
Preprocessing can be performed with the [`zq`](https://github.com/brimdata/zed/blob/main/docs/zq/README.md)
```
```suggestion
Preprocessing can be performed with the [`zq`](https://github.com/brimdata/zed/blob/main/docs/commands/zq.md)
```

Reply:
@mason-fish: Per our discussion on Slack, if the new Zed docs site is where things are converging, then instead of fixing the links to point into the `docs/` location within the GitHub repo it seems like it might make sense to instead point to the relevant page below [https://zed.brimdata.io/docs/.](https://zed.brimdata.io/docs/.) I've just pushed a commit to adjust three links in here to do that. If you agree the approach makes sense, feel free to merge, as I don't have the permissions to do that myself. Or if there's a desire to keep linking into the GItHub docs location for now, let me know and I can revert that commit and go with your suggestion instead.